lib/cppcheck.cpp

```diff
@@ -291,7 +295,8 @@ static std::string executeAddon(const AddonInfo &addonInfo,
     }
 
     const std::string fileArg = (endsWith(file, FILELIST, sizeof(FILELIST)-1) ? " --file-list " : " ") + cmdFileName(file);
-    const std::string args = cmdFileName(addonInfo.scriptFile) + " --cli" + addonInfo.args + fileArg;
+    const std::string args =
+        addonInfo.runScript + " " + cmdFileName(addonInfo.scriptFile) + " --cli" + addonInfo.args + fileArg;
```

Comment:
in case addonInfo.runScript contains spaces and stuff.. use `cmdFileName` on that also.